specs/docker/rename_docker_container.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Rename a Docker container
+command: "docker container rename {{old_container}} {{new_container}}"
+tags:
+  - docker
+description: Renames a Docker container.
+arguments:
+  - name: old_container
+    description: The name of the old container
+    default_value: ~
+  - name: new_container
+    description: The name of the new container
+    default_value: ~
```

Comment:
Same here!

I think we should remove the default value parameter, since the actual container names will be specific to each user.